rel/i18n/emqx_dashboard_api.hocon

```diff
@@ -90,4 +90,23 @@ backend.desc:
 password_expire_in_seconds.desc:
 """Time in seconds until the password expiresc."""
 
+bad_mfa_token.desc:
+"""Login failed due to MFA token is not provided or invalid."""
+
+change_mfa.desc:
+"""Setup MFA for a dashboard user."""
+
+delete_mfa.desc:
+"""Delete MFA state for a dashboard user.
+After the state is deleted, the user will have MFA explicitly disabled, to re-enabled MFA, call the `POST` method."""
+
+mfa_token.desc:
+"""Multifactor authenticaion token."""
+
+mfa_status.desc:
+"""MFA status.
+- `none`: no MFA enabled
+- `disabled`: MFA is disabled for the user
```

Comment:
Q: is there a reason to differentiate between the two?

Perhaps the intention of `disabled` is to mean "MFA state/secret is already set up, but we don't require using it for this user"?